### PR TITLE
Implemented allowConsecutiveUsings option

### LINKED ISSUE #2184: Add setting to relax SA1519 to allow consecutive with braces omitted on all but the last one
#2180 modified the rule to disallow this:

```
using (var a = GetDisposable())
using (var b = GetAnotherDisposable())
{
}
```

This was permitted in classic StyleCop. It would be helpful to have a setting that allows this behaviour to be retained.

### PR DESCRIPTION
**Pull request by vweijsters**
Fixes #2184

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1519BracesMustNotBeOmittedFromMultiLineChildStatement.cs [hunk 1]
```diff
@@ -108,9 +109,15 @@ private static void HandleForEachStatement(SyntaxNodeAnalysisContext context)
             CheckChildStatement(context, forEachStatement.Statement);
         }
 
-        private static void HandleUsingStatement(SyntaxNodeAnalysisContext context)
+        private static void HandleUsingStatement(SyntaxNodeAnalysisContext context, StyleCopSettings settings)
         {
             var usingStatement = (UsingStatementSyntax)context.Node;
+
+            if (settings.LayoutRules.AllowConsecutiveUsings && (usingStatement.Statement is UsingStatementSyntax))
```
Comment (pdelvo) [inline StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1519BracesMustNotBeOmittedFromMultiLineChildStatement.cs]: Would this allow something like this?
```csharp
using(...)
for(...)
{
// ...
}
```
What is the stylecop classic behavior on this?

File: documentation/Configuration.md [hunk 2]
```diff
@@ -313,6 +314,14 @@ file are handled. The `newlineAtEndOfFile` property supports the following value
 * `"require"`: Files are required to end with a single newline character
 * `"omit"`: Files may not end with a newline character
 
+### Consecutive using statements without braces
```
Comment (pdelvo) [inline documentation/Configuration.md]: This should probably specify if using statements are allowed without braces before other similar constructs like `for`

File: StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1519BracesMustNotBeOmittedFromMultiLineChildStatement.cs (hunk 1 above)
Comment (vweijsters) [inline StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1519BracesMustNotBeOmittedFromMultiLineChildStatement.cs]: That would not be allowed, as the code explicitly checks if the child statement is a using statement.

StyleCop classic also does not allow that construction.

File: documentation/Configuration.md (hunk 2 above)
Comment (vweijsters) [inline documentation/Configuration.md]: I've added a clarification for this

**Pull request merged**